reports_simplified.php

```diff
@@ -20,7 +20,26 @@ function reportsEscape( $text )
 	                     [ '<b>', '</b>', '<i>', '</i>' ], $text );
 	$text = preg_replace( '/&(amp;)*amp;lt;(b|i)&(amp;)*amp;gt;/', '&$1lt;$2&$3gt;', $text );
 	$text = preg_replace( '/&(amp;)*amp;lt;\\/(b|i)&(amp;)*amp;gt;/', '&$1lt;/$2&$3gt;', $text );
-	$text = str_replace( "\n", $svbr, $text );
+	$lineCount = 0;
+	$textLines = explode( "\n", $text );
+	$text = '';
+	while ( ! empty( $textLines ) )
+	{
+		if ( $text != '' )
+		{
+			if ( $lineCount > 25 )
```

Comment:
I saw similar (same?) code at the start of alerts_simplified.php.  Not sure why literal number of 25 is used, and code Deja-vu always makes me nervous of the potential of diverging updates.  Can there be a single function call instead?  especially if it appears later in code I've not reviewed yet!

Reply:
I have amended this so the line count is defined as a constant.

Reply:
You gave a nice explanation of the line break issue in excel - could that please be copied into a comment here so that next time I or someone else sees this, there is the necessary background info.

Reply:
I will add this.